### ISSUE TITLE
DSA probable PQ gen needs domainSeed from IUT

### ISSUE THREAD

**Issue by smuellerDD** [usnistgov/ACVP#405]
For DSA provable PQ gen, the server replies to my answers with:

```
}, {
  "results" : {
    "vsId" : 8380,
    "disposition" : "fail",
    "tests" : [ {
      "tcId" : 1,
      "result" : "fail",
      "reason" : "Unexpected failure"
    }, {
      "tcId" : 2,
      "result" : "fail",
      "reason" : "Unexpected failure"
```

**Comment by celic** [comment]
Just to be clear... the registration says "probable" not "provable". Did you intend this? or does this affect the issue?

**Comment by smuellerDD** [comment]
I meant to refer to probable PQ gen.

Provable PQ gen works (see vsID 4509).

**Comment by celic** [comment]
It doesn't look like you provide the server with the `domainSeed`. I believe this value is needed for the server to validate probable PQGen.

**Comment by smuellerDD** [comment]
Am Mittwoch, 20. Juni 2018, 20:03:39 CEST schrieb Chris Celi:

Hi Chris,

> It doesn't look like you provide the server with the `domainSeed`. I believe
> this value is needed for the server to validate probable PQGen.

Ok, will try that one -- the examples in the Appendix did not include the 
domainSeed and so I did not either.

Ciao
Stephan

**Comment by celic** [comment]
Then this may be a spec issue.

**Comment by smuellerDD** [comment]
Confirmed, after adding the domainSeed, the ACVP server likes it.

**Comment by smuellerDD** [comment]
Fixed with patch to issue #359
